### PR TITLE
Updated VCU test based on new softkernel changes

### PR DESCRIPTION
**Pull request by saifuddin-xilinx**

### CODE REVIEW

**Comment by xilinxgitops** [comment]
retest this please

File: tests/validate/xcl_vcu_test/src/host.cpp [hunk 1]
```diff
@@ -80,8 +80,14 @@ int main(int argc, char** argv) {
   // Harcoding the number of processes/instances 
   int ret = vcu_dec_test(binaryFile.c_str(), TEST_INSTANCE_ID, device_index);
   if (ret != TRUE) {
-    std::cout << "TEST FAILED\n";
-    return EXIT_FAILURE;
+    if (ret == FALSE) {
```
Comment (AShivangi) [inline tests/validate/xcl_vcu_test/src/host.cpp:83]: we should simplify this:
if(ret == FALSE)...
else if (ret!= not_SUPP)...

**Review by AShivangi — CHANGES_REQUESTED** [review]

File: tests/validate/xcl_vcu_test/src/host.cpp (hunk 1 above)
Comment (saifuddin-xilinx) [reply tests/validate/xcl_vcu_test/src/host.cpp]: Done.

**Review by AShivangi — APPROVED** [review]

**Review by chvamshi-xilinx — APPROVED** [review]

**Comment by chvamshi-xilinx** [comment]
@saifuddin-xilinx Please backport it to U30 branch

**Pull request merged**